languages/python/concepts/lists/about.md

```diff
@@ -0,0 +1,242 @@
+
+>>> nested_data_structures = [
+                                 {"fish": "gold", "monkey": "brown", "parrot" : "grey"},
+                                 ("fish", "mammal", "bird"),
+                                 ['water', 'jungle', 'sky']
+                             ]
+[{"fish": "gold", "monkey": "brown", "parrot" : "grey"}, ("fish", "mammal", "bird"), ['water', 'jungle', 'sky']]
+```
+
+The `list()` constructor can be used empty or with an _iterable_ as an argument. Elements in the iterable are cycled through by the constructor and added to the list in order:
+
+```python
+>>> no_elements = list()
+[]
+
+#the tuple is unpacked and each element is added
+>>> multiple_elements_tuple = list(("Parrot", "Bird", 334782))
+["Parrot", "Bird", 334782]
+
+#the set is unpacked and each element is added
+>>> multiple_elements_set = list({2, 3, 5, 7, 11})
+[2,3,5,7,11]
+```
+
+Results when using a list constructor with a string or a dict may be surprising:
+
+````python
+
+#string elements (Unicode code points) are iterated through and added *individually*
+>>> multiple_elements_string = list("Timbuktu")
+['T', 'i', 'm', 'b', 'u', 'k', 't', 'u']
+
+
+>>> multiple_code_points_string = list('अभ्यास')
+['अ', 'भ', '्', 'य', 'ा', 'स']
+
+"""
+The iteration default for dictionaries is over the keys.
+"""
+source_data = {"fish": "gold", "monkey": "brown"}
+>>> multiple_elements_dict_1 = list(source_data)
+['fish', 'monkey']
+
+Because the constructor will only take _iterables_ (or nothing) as arguments, objects that are _not_ iterable will throw a type error. Consequently, it is much easier to create a one-item list via the literal method.
+
+```python
+
+>>> one_element = list(16)
+Traceback (most recent call last):
+   File "<stdin>", line 1, in <module>
+   TypeError: 'int' object is not iterable
+
+>>> one_element_from_iterable = list((16,))
+[16]
+````
+
+## Accessing elements
+
+Items inside lists (_like the sequence types `string` and `tuple`_), can be accessed via 0-based index and _bracket notation_. Indexes can be from **`left`** --> **`right`** (_starting at zero_) or **`right`** --> **`left`** (_starting at -1_).
+
+| **0** | **1** | **2** | **3** | **4** | **5** |\
+ -------------------------\
+ | P | y | t | h | o | n |\
+ -------------------------\
+ |_**-6**_ |_**-5**_ |_**-4**_ |_**-3**_ |_**-2**_ |_**-1**_ | <----
+
+```python
+
+>>> breakfast_foods = ["Oatmeal", "Fruit Salad", "Eggs", "Toast"]
+
+#Oatmeal is at index 0 or index -4
+>>> first_breakfast_food = breakfast_foods[0]
+'Oatmeal'
+
+>>> first_breakfast_food = breakfast_foods[-4]
+'Oatmeal'
+```
+
+The method `.pop()` can be used to both remove and return a value at a given index:
+
+```python
+
+>>> breakfast_foods = ["Oatmeal", "Fruit Salad", "Eggs", "Toast"]
+
+#Fruit Salad is at index 1 or index -3
```

Comment:
Same for all comments in code blocks:
"Comments should start with a # and a single space." Pep8